jnigen/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.6.0-wip.3
+* **Breaking Change** ([#354](https://github.com/dart-lang/jnigen/issues/354)): Renamed constructors from `ctor1`, `ctor2`, ... to `new1`, `new2`, ...
```

Comment:
In the generated files I see `new0`?

Reply:
`_new0` is the private name for unnamed constructor. It won't be visible to the users.